### ISSUE TITLE
New firmware corrupt

### ISSUE THREAD

**Issue by emudem** [QIDITECH/QIDI_PLUS3#7]
The version 4.2.12 is corrupt.

En español.

Al terminar la instalación de la versión 4.2.12, la verificación da error dejando la impresora brickeada.

**Comment by emudem** [comment]
Segun after sales...

Dear Gracias,  

In that case, we need to insert the mini Sd card directly into the screen to do the update. 

Do you have min sd card in your place ? If you do, i will send you the another update file, it will works. 


Best regards,
Lisa

**Comment by QIDITECH** [comment]
Yes, you need to save the repaired file to the mini sd card and then insert it into the card slot of the screen.
Github does not support this format file, so we cannot serve you on Github.
